assets/training/model_management/components/convert_model_to_mlflow/spec.yaml

```diff
@@ -28,6 +28,15 @@ inputs:
     description: Huggingface model id (https://huggingface.co/<model_id>). A required parameter for Huggingface model framework. Can be provided as input here or in model_download_metadata JSON file.
     optional: true
 
+  model_flavor:
+    type: string
+    enum:
+      - HFTransformersV2
+      - OSS
+    default: HFTransformersV2
+    optional: false
```

Comment:
we should also think about making it optional if not including pyfunc as favor. For vision models this property would not be used